include/deal.II/distributed/tria_base.h

```diff
@@ -246,6 +246,27 @@ namespace parallel
   template <int dim, int spacedim = dim>
   using Triangulation DEAL_II_DEPRECATED = TriangulationBase<dim, spacedim>;
 
+  template <int dim, int spacedim = dim>
+  class DistributedTriangulationBase
+    : public dealii::parallel::TriangulationBase<dim, spacedim>
```

Comment:
This class is missing documentation. Can you add something short that this is an intermediate class that can be used to test (via `dynamic_cast`) whether a particular triangulation object is able to distribute meshes between processors?

Reply:
Done!